tests/test_terraform.py

```diff
@@ -138,3 +139,34 @@ def test_hello_world(hello):
 
     # make sure that that we get a '0' exit code for the testsuite
     assert result.ret == 0
+
+
+@pytest.mark.skipif(not tf.find_binary("terraform"), reason="Terraform binary missing")
+def test_plugins_ini_setting_terraform_mod_dir(testdir):
+    mod_dir = Path(__file__).parent / "mrofarret"
```

Comment:
nit. can you stick this in a data subdir off the test dir just to keep it obvious to purpose, plus we may have others.

Reply:
can do!